src/components/alert/Alert.tsx

```diff
@@ -0,0 +1,9 @@
+import { Alert as AntdAlert, AlertProps } from 'antd'
+
+export type Props = AlertProps
```

Comment:
Could we use `Expand` for all of these component types? It should help consumers of the design system to know what the types of these props are better